src/themes.php

```diff
@@ -699,4 +699,17 @@
         "sideNums" => "#F8D866",
         "stroke" => "#4A4F64",
     ],
+    "city-lights" => [
+        "background" => "#1D252CFF",
+        "border" => "#1D252CFF",
+        "currStreakLabel" => "#5D8CB3FF",
+        "currStreakNum" => "#5D8CB3FF",
+        "dates" => "#5D8CB3FF",
+        "fire" => "#718CA1FF",
+        "hide_border" => "true",
+        "ring" => "#718CA1FF",
+        "sideLabels" => "#5D8CB3FF",
+        "sideNums" => "#61788AFF",
+        "stroke" => "#536676FF",
+    ]
```

Comment:
`hide_border` is not one of the fields for themes, but you could use `"border" => "#0000"` to make the border transparent by default. Unless it's necessary, I think it would be best to have a normal border and allow users to add `&hide_border=true` to their cards if they choose to.

The `FF` at the end of each color is not required since it represents full opacity. It is simpler to just remove it if you're not using transparency.

```suggestion
    "city-lights" => [
        "background" => "#1D252C",
        "border" => "#1D252C",
        "currStreakLabel" => "#5D8CB3",
        "currStreakNum" => "#5D8CB3",
        "dates" => "#5D8CB3",
        "fire" => "#718CA1",
        "ring" => "#718CA1",
        "sideLabels" => "#5D8CB3",
        "sideNums" => "#61788A",
        "stroke" => "#536676",
    ],
```